docs/QbeastFormat.md

```diff
@@ -18,12 +18,12 @@ Following each write transaction is the creation of a new log file. **Table-leve
 
 ## Qbeast-spark on Delta
 
-Qbeast format extends the Delta Lake Format, so the data is stored in Parquet
+Qbeast format **extends the Delta Lake Format**, so the data is stored in Parquet
 files and the Index Metadata is stored in the Delta log. 
```

Comment:
Either give an example or point to another location that has an example. I recall seeing the schema of an addFile entry somewhere in the documentation.

Reply:
Is in the same document 👍